stock_request_mrp/tests/test_stock_request_mrp.py

```diff
@@ -92,13 +92,11 @@ def test_create_request_01(self):
         self.assertEqual(
             manufacturing_order.company_id, order.stock_request_ids[0].company_id
         )
-        manufacturing_order.action_confirm()
-        manufacturing_order.write(
-            {"qty_producing": manufacturing_order.product_uom_qty}
+        res = manufacturing_order.button_mark_done()
```

Comment:
Why putting the production as done? That's different from confirming it, which was the previous action.

Reply:
Mark as done to finalize and check `qty_done` (same as in v15: https://github.com/OCA/stock-logistics-warehouse/blob/15.0/stock_request_mrp/tests/test_stock_request_mrp.py#L101).

Reply:
It is not necessary to confirm it (in v15 it is not really necessary either); however, I have added in the test to check that the production is confirmed.